Free page  appears after updating IAM Policy

server deploy: windows10 docker
run enviroment: google chrome
google chrome console error:
Error: Minified React error #31;  react-dom.production.min.js:216 Error: Minified React error #31; 
visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=TypeError%3A%20r%20is%20not%20a%20function for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
at Oi (react-dom.production.min.js:140)
    at react-dom.production.min.js:150
    at Ia (react-dom.production.min.js:176)
    at Gs (react-dom.production.min.js:271)
    at jl (react-dom.production.min.js:250)
    at kl (react-dom.production.min.js:250)
    at _l (react-dom.production.min.js:250)
    at vl (react-dom.production.min.js:243)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:18)

![image](https://user-images.githubusercontent.com/26241803/124531595-68f41800-de41-11eb-9211-93df9910a83a.png)

--- a/portal-ui/src/screens/Console/Policies/PolicyDetails.tsx
+++ b/portal-ui/src/screens/Console/Policies/PolicyDetails.tsx
@@ -30,7 +30,7 @@ import TableWrapper from "../Common/TableWrapper/TableWrapper";
 import api from "../../../common/api";
 import PageHeader from "../Common/PageHeader/PageHeader";
 import { Link } from "react-router-dom";
-import { setErrorSnackMessage } from "../../../actions";
+import { setErrorSnackMessage, setSnackBarMessage } from "../../../actions";
 import CodeMirrorWrapper from "../Common/FormComponents/CodeMirrorWrapper/CodeMirrorWrapper";
 import history from "../../../history";
 import InputAdornment from "@material-ui/core/InputAdornment";
@@ -44,8 +44,8 @@ import TextField from "@material-ui/core/TextField";
 interface IPolicyDetailsProps {
   classes: any;
   match: any;
-  closeModalAndRefresh: (refresh: boolean) => void;
   setErrorSnackMessage: typeof setErrorSnackMessage;
+  setSnackBarMessage: typeof setSnackBarMessage;
 }
 
 const styles = (theme: Theme) =>
@@ -160,8 +160,8 @@ const styles = (theme: Theme) =>
 const PolicyDetails = ({
   classes,
   match,
-  closeModalAndRefresh,
   setErrorSnackMessage,
+  setSnackBarMessage,
 }: IPolicyDetailsProps) => {
   const [selectedTab, setSelectedTab] = useState<number>(0);
   const [policy, setPolicy] = useState<Policy | null>(null);
@@ -191,7 +191,7 @@ const PolicyDetails = ({
       })
       .then((res) => {
         setAddLoading(false);
-        closeModalAndRefresh(true);
+        setSnackBarMessage("Policy successfully updated")
       })
       .catch((err) => {
         setAddLoading(false);
@@ -441,6 +441,7 @@ const PolicyDetails = ({
 
 const connector = connect(null, {
   setErrorSnackMessage,
+  setSnackBarMessage
 });
 
 export default withStyles(styles)(connector(PolicyDetails));

